handlers/homu_status/__init__.py

```diff
@@ -5,45 +5,84 @@
 
 from eventhandler import EventHandler
 
+WPT_RESULT_ARROW = '\xe2\x96\xb6'
 
-def check_failure_log(api, bors_comment):
-    # bors_comment would be something like,
-    # ":broken_heart: Test failed - [linux2](http://build.servo.org/builders/linux2/builds/2627)"  # noqa
-    # ... from which we get the relevant build result url
-    url = iter(re.findall(r'.*\((.*)\)', str(bors_comment))).next()
-    if not url:
-        return
 
-    # Substitute and get the new url
-    # (e.g. http://build.servo.org/json/builders/linux2/builds/2627)
-    json_url = re.sub(r'(.*)(builders/.*)', r'\1json/\2', url)
-    json_stuff = api.get_page_content(json_url)
-    if not json_stuff:
-        return
+def parse_log(log):     # gets the names of the failed tests
+    test_names = []
+    l = log.split()
```

Comment:
I didn't go for the error summary log, because we already have the entire log on hand, and we're confident about the format. Wouldn't it be better if we avoid additional network requests?